hub/util/keys.py

```diff
@@ -16,5 +22,20 @@ def get_tensor_meta_key(key: str) -> str:
     return posixpath.join(key, constants.TENSOR_META_FILENAME)
 
 
-def get_index_meta_key(key: str) -> str:
-    return posixpath.join(key, constants.INDEX_META_FILENAME)
+def get_chunk_id_encoder_key(key: str) -> str:
+    return posixpath.join(
+        key,
+        constants.ENCODED_CHUNK_NAMES_FOLDER,
+        constants.ENCODED_CHUNK_NAMES_FILENAME,
+    )
+
+
+"""existence methods"""
```

Comment:
Not a fan of these comments. All the methods have their section in the title, that should be plenty.